composer.json

```diff
@@ -13,21 +13,35 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.5.0",
```

Comment:
imho it's better to follow semver: `"^5.5.0 || ^7.0.0"

Reply:
Good point, thanks! a1b17cf